test/unit/engine_runtime.js

```diff
@@ -239,3 +239,14 @@ test('setCompatibilityMode does not restart if it was not running', t => {
     t.equal(started, false);
     t.end();
 });
+
+test('Disposing the runtime emits an event', t => {
+    let disposed = false;
+    const rt = new Runtime();
+    rt.addListener('RUNTIME_DISPOSED', () => {
+        disposed = true;
+    });
+    rt.start();
```

Comment:
```suggestion
    rt.dispose();
```

Reply:
I tried to commit the above suggestion but turns out I don't have permission to push to your fork...